docs/blog/2019-04-04-how-to-build-a-blog-with-wordpress-and-gatsby-part-2/index.md

```diff
@@ -0,0 +1,408 @@
+    "prop-types": "^15.7.2",
+    "react": "^16.8.6",
+    "react-dom": "^16.8.6",
+    "react-helmet": "^5.2.0"
+  },
+  "devDependencies": {
+    "prettier": "^1.16.4"
+  },
+  "keywords": [
+    "gatsby"
+  ],
+  "license": "MIT",
+  "scripts": {
+    "build": "gatsby build",
+    "develop": "gatsby develop",
+    "format": "prettier --write src/**/*.{js,jsx}",
+    "start": "npm run develop",
+    "serve": "gatsby serve",
+    "test": "echo \"Write tests! -> https://gatsby.dev/unit-testing\""
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/gatsbyjs/gatsby-starter-default"
+  },
+  "bugs": {
+    "url": "https://github.com/gatsbyjs/gatsby/issues"
+  }
+}
+```
+
+You can see there are several dependencies installed right off the bat. I'll just cover a few of them. Gatsby-Image creates an effect similar to Medium and also allows you to use responsive images and optimize your site. Gatsby-Transformer-Sharp is what creates the responsive and optimized images, then allows you to query for those through GraphQL. You also have Gatsby-Source-Filesystem which could be used to pull in markdown files for content, but you're using WordPress instead. The last package I want to mention is Gatsby-Plugin-React-Helmet, which allows you to create meta tags for the site's head which helps with Search Engine Optimization, or SEO.
+
+Whew! That was a mouthful.
+
+<video autoplay muted loop width="400">
+  <source src="https://media.giphy.com/media/eb3WAhXzlUAFi/giphy.mp4">
+</video>
+
+### Running the site
+
+You will be running your Gatsby.js site in development so you can see what you're doing. Kinda hard to fly a plane when you can't see where you're going, right?
+
+To do this, run the following command in the terminal and it will build the site in a development environment with hot reloading and more.
+
+```shell
+gatsby develop
+```
+
+After running that command, you can visit [localhost:8000](http://localhost:8000) in the browser and you should see the site pictured below:
+
+![Initial appearance of Gatsby.js starter](images/gatsby1.png)
+
+The site provides a navbar with a link going back to the homepage. There is also a bit of content with a link to page 2 which then provides a link back to page 1. It's a very simple site, but already you can see how fast Gatsby.js is.
+
+## Adding Gatsby.js plugins
+
+Now that you know what's already installed and what it looks like to start, you can add the stuff you need to use WordPress with your site. Fortunately, Gatsby has a page on their site where you can [see what plugins are available](https://www.gatsbyjs.org/plugins/). You will be adding the following plugins to your site: [Gatsby-Source-WordPress](https://www.gatsbyjs.org/packages/gatsby-source-wordpress/) and [Gatsby-Plugin-Sitemap](https://www.gatsbyjs.org/packages/gatsby-plugin-sitemap/).
+
+To do this, you can use this code in the terminal:
+```shell
+yarn add gatsby-source-wordpress gatsby-plugin-sitemap
```

Comment:
We recommend documenting (and a blog is documentation, of sorts!) using `npm` if at all possible.

The idea being that anyone who uses `yarn` knows they can swap the commands, whereas the same may not be true of someone newer to the ecosystem--which would just introduce confusion!

Reply:
@DSchau Ah, that makes sense. I'll update any shell commands using yarn to use npm instead.

Reply:
Changed yarn commands over to npm